documentation/bolt_types_reference.md

```diff
@@ -2,7 +2,7 @@
 
 This page lists custom data types used in Bolt plans and their functions.
 
-## `ApplyResult`
+### `ApplyResult`
```

Comment:
```suggestion
## Data types

Bolt ships with several data types that can be used within a plan.

### `ApplyResult`
```

There needs to be a H2 header before this, otherwise the dita build will fail.